public/app/features/alerting/unified/components/admin/ExternalAlertmanagers.tsx

```diff
@@ -17,15 +17,14 @@ import {
 import EmptyListCTA from 'app/core/components/EmptyListCTA/EmptyListCTA';
 import { loadDataSources } from 'app/features/datasources/state/actions';
 import { AlertmanagerChoice } from 'app/plugins/datasource/alertmanager/types';
-import { useDispatch, useSelector } from 'app/types';
-import { StoreState } from 'app/types/store';
+import { useDispatch } from 'app/types';
 
+import { alertmanagerApi } from '../../api/alertmanagerApi';
 import { useExternalAmSelector, useExternalDataSourceAlertmanagers } from '../../hooks/useExternalAmSelector';
-import {
-  addExternalAlertmanagersAction,
-  fetchExternalAlertmanagersAction,
-  fetchExternalAlertmanagersConfigAction,
-} from '../../state/actions';
+import // addExternalAlertmanagersAction,
+// fetchExternalAlertmanagersAction,
+// fetchExternalAlertmanagersConfigAction,
```

Comment:
Are these comments on purpose or should they be removed?

Reply:
Should be removed :)